source/topics/paas/how-to-automate-database-migrations.md

```diff
@@ -7,4 +7,25 @@ For example, to automate Rails migrations, you might have the following `.aptibl
 ```yaml
 before_release:
   bundle exec rake db:migrate
-```
\ No newline at end of file
+```
+
+If you want to deploy the same repo to different environments that require different rake tasks, we suggest referencing a bash script:
```

Comment:
Pedantic: "bash" –> "Bash" (generally, refer to official [docs](https://www.gnu.org/software/bash/) as a spelling/capitalization guide.